website/docs/reference/project-configs/log-path.md

```diff
@@ -0,0 +1,19 @@
+---
+datatype: directorypath
+default_value: logs
+---
+# Definition
+Optionally specify a custom directory where logs are located.
```

Comment:
Can probably write something a little nicer here, along the lines of:
> When dbt runs, it generates logs. These logs are written to whichever directory you specify. 

(but better grammar)